src/main/webapp/app/exercises/programming/manage/grading/programming-exercise-configure-grading.component.html

```diff
@@ -256,5 +259,12 @@ <h2 class="mb-5 fw-medium">
                 </div>
             }
         </div>
+        <div class="grading-body-container mt-3">
+            @if (activeTab === 'test-analysis') {
+                <div>
+                    <jhi-testcase-analysis [exerciseTitle]="programmingExercise.title" />
+                </div>
```

Comment:
Can you check if we need to wrap it in an additional div here? It looks like we could leave it out:
```suggestion
                <jhi-testcase-analysis [exerciseTitle]="programmingExercise.title" />
```